dietpi/dietpi-software

```diff
@@ -7079,6 +7085,82 @@ _EOF_
 			/boot/dietpi/func/create_mysql_db wordpress wordpress "$GLOBAL_PW"
 		fi
 
+		if To_Install 141 # Hugo
+		then
+			aDEPS=("git")
```

Comment:
We have an install option for Git (ID 17). Should we use this one?

Reply:
oh right, I think I never quite understood the different between `aDEPS` and `aSOFTWARE_DEPS[$software_id]`

Reply:
Yes, use
```sh
aSOFTWARE_DEPS[$software_id]='17'
```
in the first code block instead. But, actually I think Git is not required. It is only used to clone the PaperMod theme, while you could simply download the archive directly: https://github.com/adityatelange/hugo-PaperMod/archive/master.tar.gz